solana-programs/payment-router/tests/payment-router.ts

```diff
@@ -28,6 +28,7 @@ const SPL_AUDIO_TOKEN_ADDRESS_KEY = new PublicKey(SPL_AUDIO_TOKEN_ADDRESS)
 const SPL_USDC_TOKEN_ADDRESS_KEY = new PublicKey(SPL_USDC_TOKEN_ADDRESS)
 
 const endpoint = 'https://api.mainnet-beta.solana.com'
+// const endpoint = 'http://localhost:8899'
```

Comment:
could this come from the config? is it needed? or maybe add a comment?

Reply:
I can add a comment. I was planning to follow up and move this into an env var which defaults to mainnet